apps/molgenis-components/src/components/forms/formUtils/formUtils.test.ts

```diff
@@ -170,6 +170,17 @@ describe("getRowErrors", () => {
     expect(result).to.deep.equal({});
   });
 
+  // test("it should return no error for valid localhost hyperlink", () => {
```

Comment:
so this test should be re-enabled?

Reply:
I'm not sure if we needed it anymore. I'll remove it